### PR TITLE
feat: 2789 Non team queues

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: cases/views/search/views.py [hunk 1]
```diff
@@ -59,7 +61,10 @@ def get(self, request, *args, **kwargs):
         service.populate_is_recently_updated(cases)
         service.get_hmrc_sla_hours(cases)
 
-        queue = next((q for q in queues if str(q["id"]) == str(queue_id)))
+        queue = (
+            next((q for q in queues if str(q["id"]) == str(queue_id)), None)
+            or list(Queue.objects.filter(id=queue_id).annotate(case_count=Count("cases")).values())[0]
```
Comment (andytaylorkainos) [inline cases/views/search/views.py:66]: list(Queue.objects.filter(id=queue_id).annotate(case_count=Count("cases")).first())

File: cases/views/search/views.py [hunk 2]
```diff
@@ -59,7 +61,10 @@ def get(self, request, *args, **kwargs):
         service.populate_is_recently_updated(cases)
         service.get_hmrc_sla_hours(cases)
 
-        queue = next((q for q in queues if str(q["id"]) == str(queue_id)))
+        queue = (
```
Comment (efahy) [inline cases/views/search/views.py:64]: Comment please, indicating that that the user may be on a queue that is neither a system queue or one of their team's queues, in which case we need to go to the DB again to get it

Comment (efahy) [reply cases/views/search/views.py:64]: Also, should we consider an API test for this scenario?

Comment (BenAndrew-Kainos) [reply cases/views/search/views.py:66]: Done

**Pull request merged**